include/logging/log.h

```diff
@@ -21,11 +21,11 @@ extern "C" {
  * @{
  */
 
-#define LOG_LEVEL_NONE  0
-#define LOG_LEVEL_ERR   1
-#define LOG_LEVEL_WRN   2
-#define LOG_LEVEL_INF   3
-#define LOG_LEVEL_DBG   4
+#define LOG_LEVEL_NONE _LOG_LEVEL_NONE
```

Comment:
A curious kid in me really wonders what happens here, but nevermind.

Reply:
I.e. why we can't just define the final non-underscore version in log_core.h and be done with it.

Reply:
removed that change. it was not needed.